src/tools/genpolicy/rules.rego

```diff
@@ -883,72 +882,114 @@ mount_source_allows(p_mount, i_mount, bundle_id, sandbox_id) {
 
     print("mount_source_allows 2: true")
 }
+mount_source_allows(p_mount, i_mount, bundle_id, sandbox_id) {
+    print("mount_source_allows 3: i_mount.source=", i_mount.source)
+
+    i_source_parts = split(i_mount.source, "/")
```

Comment:
Can we change the format of the Input from the Shim to not include a Guest path? 

Ideally, the Shim should specify just: 
- An ID of the storage destination mount, or
- At most, a simple mount directory name.

Either one of these should have a well-defined format, that can be easily verified. 

The Agent can then add the rest of the path, as needed. Then this new Policy function would become simpler and safer for the confidentiality of the Guest.